source/_patterns/02-molecules/card/card.twig

```diff
@@ -60,16 +95,24 @@
         </div>
       {% endif %}
 
-      {% if (style == 'cta' or style == 'media-left') and card.link.url %}
-        <div class="usa-card__footer">
-          <a class="{{ card.link.type }}" href="{{ card.link.url }}">{{ card.link.text }}</a>
-        </div>
+      {% if ('cta' in style or 'media' in style) and card.link.url %}
+        {% if card.link.text %}
+          <div class="usa-card__footer">
+            <a class="{{ card.link.type }}" href="{{ card.link.url }}">{{ card.link.text }}</a>
+          </div>
+        {% endif %}
       {% endif %}
     </div>
 
     {% if card.media %}
       <div class="usa-card__media">
+      {% if not card.media.src %}
```

Comment:
This is to account for C3's having an icon that shows up on the left as an SVG.  They aren't using the card.media.src but are embedding it straight into card.media so I'm checking here for that option explicitly so as to not break local theme.

Reply:
I was thinking the context order which should be default. Could be card.media since it's more flexible to accept any media - SVG or IMG - while overlay is only IMG. Not critical or required.

Reply:
Doesn't card.media result in true in either way?  I'm trying to see if any src file location or URL is passed and if so that would take precedence in the event that somehow both media and media.src have values passed. Right?

Reply:
Yes but also they are different use case/options that have been creatively combined under one variable. Is that difference clear in the code or maybe my other comment on noting the differences might be more useful at this point (if that is even needed).

Again, not an issue but just a suggested context. We can resolve since it's not needed.

Reply:
Yeah, the problem is ideally we would treat media.src as the location of the icon or image but that was embedded HTML used in card.media previously so if I changed that, it could mess up things on TCs because it wouldn't pass as media.src without us having to update Drupal.  I'll comment it for now and we'll have to revisit and standardize on one approach later, I think.